### ISSUE TITLE
Congo vs Dem. Republic of the Congo

### ISSUE THREAD

**Issue by Tyto1984** [teuteuf/worldle#325]
“Congo” was accepted as answer, but was 784 km off. The correct answer was “Democratic Republic of the Congo”. 
![9F914E25-349D-4565-9F41-312F420F01F7](https://user-images.githubusercontent.com/114105010/191584614-3c6085a7-9045-4ac3-bcd4-3155972f4898.png)

**Comment by DavidLloydOwen** [comment]
The Republic of the Congo and the Democratic Republic of the Congo are distinct and separate countries. They share a border with the DRC lying to the east of the smaller RoC. Does this resolve the issue you raised?

**Comment by Tyto1984** [comment]
Not really, because both are referred to as "Congo". Since Congo refers to both, it should not have been accepted as answer.

**Comment by marco-schmidt** [comment]
I requested the two Congos to be distinguishable a while ago: https://github.com/teuteuf/worldle/issues/237
